memory/README.md

```diff
@@ -0,0 +1,5 @@
+# CoreKV Memory store
+
+A lightweight in-memory CoreKV Store implementation.
+
+Prioritizes memory consumption over speed, it's main use-case is for versioned queries within Defra. Other, external, use-cases are currently supported, but are not targeted (this may change).
```

Comment:
thought: Not sure that this is accurate. The main difference in speed between the memory store and the badger memory store was the instantiation overhead for every version query. So speed was the main reason for it. It also used less memory.

Reply:
Good shout, thanks Fred - I've tweaked the first sentence to note this